glide.yaml

```diff
@@ -1,7 +1,7 @@
 package: github.com/uber/prototool
 import:
   - package: github.com/emicklei/proto
-    version: 0cbee195654dd0aab66561d51e08d37554fd8cfe
+    version: 43f8fb7abe35485d1ca39fe380c5b6d39aa931c2
```

Comment:
It's unfortunate that we need to pin to a direct hash here. We should try and get SemVer tags to depend on sometime in the future.

Reply:
Yea, but it's just how it is for now - emicklei/proto makes breaking changes, even though it does releases. See #1.

There's no real good alternative to this for now.